spectral/package.json

```diff
@@ -14,8 +14,8 @@
     "cli-integration-tests": "./scripts/rules.sh"
   },
   "dependencies": {
-    "@stoplight/spectral-cli": "^6.4.1",
-    "@stoplight/spectral-core": "^1.12.3"
+    "@stoplight/spectral-cli": "~6.4.1",
+    "@stoplight/spectral-core": "~1.12.3"
```

Comment:
What is the point of peerDependencies if they are in dependencies
```suggestion
```

Reply:
am I wrong about these? if the `~` is specified the last number ( for ~1.2.3 -> will be 3, for ~1.2 -> will be 2) will not be a hard requirement. this allows a patch increase to the dependancy which, if semver is properly being followed, should not mean a breaking change but a safer set of dependancies as patches are usually bug fixes right?
https://github.com/npm/node-semver#tilde-ranges-123-12-1

Reply:
https://docs.renovatebot.com/dependency-pinning/#why-pin-dependencies

Reply:
sold on pinning 😄